### ISSUE TITLE
`Export-PuTTYSession`: Unable to find type `[ItemNotFoundException]`

### ISSUE THREAD

**Issue by bentoner** [ralish/PSPuTTYCfg#1]
Hi Samuel,

When running
```powershell
$Sessions = Import-PuTTYSession -Path "..."
$Sessions | Export-PuTTYSession -Registry
```
I get this error
```
[91mInvalidOperation: [0mC:\Users\ben.toner\Documents\PowerShell\Modules\PSPuTTYCfg\0.2.1\PSPuTTYCfg.psm1:827
[96mLine |
[96m 827 | [0m     } catch [96m[ItemNotFoundException][0m {
[96m     | [91m             ~~~~~~~~~~~~~~~~~~~~~~~
[91m[96m     | [91mUnable to find type [ItemNotFoundException].
[0m
```
Changing "} catch [ItemNotFoundException] {" to "} catch {" fixes it.

**Comment by ralish** [comment]
Nice catch (or not, as is the case). That exception type isn't fully qualified but should be. Is fixed on `main`.

**Comment by ralish** [comment]
Fixed in [v0.2.2](https://www.powershellgallery.com/packages/PSPuTTYCfg/0.2.2).
